app/main.py

```diff
@@ -1,3 +1,31 @@
-def shop_trip():
-    # write your code here
-    pass
+import json
```

Comment:
The code uses single quotes in some places (lines 24 and 25) and double quotes in others (lines 10, 13). According to the checklist, you should use one style of quotes in your code, with double quotes being preferable.